src/css/components/_control.scss

```diff
@@ -1,4 +1,5 @@
-.video-js .vjs-control {
+.video-js .vjs-control,
+.video-js .vjs-button {
```

Comment:
Cool, got it. That makes sense but we should do that in the same PR where we adjust the buttons.